cpp/velox/compute/VeloxBackend.cc

```diff
@@ -33,16 +33,39 @@ using namespace facebook;
 
 namespace gluten {
 
+namespace {
+// Spark configs
+const std::string kSparkOffHeapMemory = "spark.gluten.memory.offHeap.size.in.bytes";
```

Comment:
Can you add the config to https://github.com/oap-project/gluten/blob/main/cpp/core/config/GlutenConfig.h#L24, then we track which setting is used in cpp side

Reply:
done